src/function/arithmetic/round.js

```diff
@@ -67,9 +71,21 @@ export const createRound = /* #__PURE__ */ factory(name, dependencies, ({ typed,
    * @return {number | BigNumber | Fraction | Complex | Array | Matrix} Rounded value
    */
   return typed(name, {
-    number: roundNumber,
+    number: function (x) {
+      if (nearlyEqual(x, roundNumber(x, epsilonExponent), config.epsilon)) {
```

Comment:
Can you reduce the double calculation by extracting `roundNumber(x, epsilonExponent)` into a variable like `const xRounded = ...`?

Reply:
(and the same for the other three function variants)

Reply:
You could also do something like:

```js
const xEpsilon = roundNumber(x, epsilonExponent)
const xSelected = nearlyEqual(x, xEpsilon, config.epsilon) ? xEpsilon : x
return roundNumber(xSelected)
```

Though I'm not sure if that is more clear or not. Up to you 😁